pkg/utils/test_util.go

```diff
@@ -37,6 +47,29 @@ func GetEventString(object runtime.Object, eventtype, reason, messageFmt string,
 			object.GetObjectKind().GroupVersionKind().Kind, object.GetObjectKind().GroupVersionKind().GroupVersion())
 }
 
+// GetObjectFromRawExtension returns an object decoded from the raw byte array
```

Comment:
Had to copy these methods from the test package cause I was not able to import them form a *_test.go file https://github.com/Azure/fleet/blob/main/test/integration/utils_test.go#L62-L78

Reply:
https://github.com/golang/go/issues/39565

Reply:
why not remove the dup from the utils_test?

Reply:
Moved methods from util_test.go